build/parser.js

```diff
@@ -914,6 +919,26 @@ class Parser {
     }
   }
 
+  /**
+   * Add:
+   * - relic data
+   * - vaulted data (probably use this over ogg)
+   * - market data on relics (urlName, id)
+   * @param {Item} item to have relics applied to
+   * @param {Array<module:warframe-relics-data.TitaniaRelic>} relics relic array to search
+   */
+  addRelics(item, relics) {
+    if (item.type !== 'Relic') return;
+    const related = relics.filter((relic) => item.name.includes(relic.name));
+    item.locations = Array.from(new Set(related.map((relic) => relic.locations).flat()));
+    item.rewards = Array.from(new Set(related.map((relic) => relic.rewards).flat()));
+    [item.marketInfo] = Array.from(new Set(related.map((relic) => relic.warframeMarket)));
+
+    const [vaultInfo] = Array.from(new Set(related.map((relic) => relic.vaultInfo)));
+    item.vaulted = vaultInfo?.vaulted;
+    item.vaultDate = vaultInfo?.vaultDate;
+  }
```

Comment:
All this set creation is for future proffing? Because there's no other need for it, the property `name`, in the generated data from titania project relics, are all unique.

Reply:
Yeah, just in case something gets in that shouldn't be in and makes it non-unique.